sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 12.1.0 (2020.02)
+- Updated Azure Storage Service API version to 2019-07-07.
+- Add support for Encryptions Scopes, which is similar to regular CPK, except the encryption key is provided by name.
```

Comment:
minor: is it officially declared as 'Encryptions Scopes', please double check plurality is by design for both encryption and scope.